common/defs-flags.go

```diff
@@ -63,12 +63,13 @@ const (
 	FlagUpdateIDDefinition         = "UpdateID to check the result of an update (either UpdateID or Update handler name should be passed)"
 	FlagCancelWorkflow             = "Cancel Workflow Execution with given Workflow Id."
 	FlagWorkflowIDTerminate        = "Terminate Workflow Execution with given Workflow Id."
+	FlagRPSDefinition              = "Limit for batch operation's requests per second."
 	FlagQueryCancel                = "Cancel Workflow Executions with given List Filter."
 	FlagQueryDelete                = "Delete Workflow Executions with given List Filter."
-	FlagQuerySignal                = "Signal Workflow Executions with given List List Filter."
+	FlagQuerySignal                = "Signal Workflow Executions with given List Filter."
+	FlagQueryReset                 = "Reset Workflow Executions with given List Filter"
 	FlagQueryTerminate             = "Terminate Workflow Executions with given List Filter."
 	FlagEventIDDefinition          = "The Event Id for any Event after WorkflowTaskStarted you want to reset to (exclusive). It can be WorkflowTaskCompleted, WorkflowTaskFailed or others."
-	FlagQueryResetBatch            = "Visibility Query of Search Attributes describing the Workflow Executions to reset. See https://docs.temporal.io/docs/tctl/workflow/list#--query."
```

Comment:
To align it with the other flags, I changed this into `FlagQueryReset` and standardized the text.